plugins/search/providers/elastic_search/lib/model/search/kBaseESearch.php

```diff
@@ -130,7 +130,7 @@ public static function getSplitIndexNamePerPartner($indexName, $partnerId)
 		if(!empty($elasticSplitIndexMap) && isset($elasticSplitIndexMap[$indexName]))
 		{
 			$splitFactor = $elasticSplitIndexMap[$indexName];
-			$index = abs(($partnerId / 10) % $splitFactor);
+			$index = abs(intval($partnerId / 10) % $splitFactor);
```

Comment:
why this is needed?

Reply:
To avoid float precision lost notices in the code